### DISCUSSION TITLE
DAG cannot execute

### DISCUSSION THREAD

**Discussion by ZJ577624359** [apache/airflow#29369] [Q&A]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

version 2.4
*** Falling back to local log
*** Log file does not exist: /home/bzwhpa/airflow/logs/dag_id=out_umu_employee_upd/run_id=manual__2023-02-02T12:09:04.963057+00:00/task_id=call_sp_umu_employee_upd/attempt=1.log
*** Fetching from: http://:8793/dag_id=out_umu_employee_upd/run_id=manual__2023-02-02T12:09:04.963057+00:00/task_id=call_sp_umu_employee_upd/attempt=1.log
*** Failed to fetch log file from worker. Request URL is missing an 'http://' or 'https://' protocol.

I configured dual-node and remote information. Because the server was restarted, the current task reported an error

![img_v2_90decc45-982b-44ca-8f6e-4c1dc6655a4g](https://user-images.githubusercontent.com/79566794/216351431-29c28902-bdd8-416b-9d3f-952bee023856.jpg)

The port and service have been checked

Thank you for providing ideas

### Operating System

linux

### Deployment

Virtualenv installation

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by hussein-awala** [comment]
The task is not scheduled or you just have this error about the log?

**Reply by ZJ577624359** [reply]
The problem has been solved
The main reason is that the 8793 port is occupied, so the task cannot be called correctly. Just query the 8793 port and kill the corresponding process
